src/REPLMode/argument_parsers.jl

```diff
@@ -33,6 +38,7 @@ end
 PackageToken(word::String)::PackageToken =
     first(word) == '@' ? VersionRange(word[2:end]) :
     first(word) == '#' ? Rev(word[2:end]) :
+    first(word) == '[' && last(word) == ']' ? Subdir(word[2:end-1]) :
```

Comment:
What is the preferred unicode-correct way to do this?

Reply:
`chop(word; head=1, tail=1)`

Reply:
Thanks.